src/Styles/index.css

```diff
@@ -11,7 +12,7 @@
     justify-content: center;
     align-items: center;
     gap: 7%;
```

Comment:
I wouldn't set it as a percentage. It is better to use rem or em. Percentages in general often cause unnecessary problems